### PR TITLE
Refactors and Fixes Wiki and Revision pages

### PR DESCRIPTION
**Pull request by Royal-lobster**
Fixes https://github.com/EveripediaNetwork/issues/issues/431

# Changes
1. made wiki main content and insights bar to span all the way till actions bar which fixes content squeezing bug
2. refactored wiki page mobile view to make code dry
3. extracted wiki page structure to ```WikiMarkup``` component which is common to revision and wiki pages 
4. removed widths for wiki main content and insights bar and replaced it with paddings and min-max widths to make implementation simpler

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
image

not sure if it's related to this PR, but I got this for the wiki with media.

ref. link: https://monopedia-orioblm3b-prediqt.vercel.app/wiki/remember-you-will-have-to-manually-add-that

**Review by Simplypheyie96 — COMMENTED** [review]
Everything looks good except for the issue @Emmanueldmlr pointed out about the media thing.

File: src/components/Wiki/WikiPage/WikiMarkup.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                }}
+                textAlign="center"
+                px={4}
+              >
+                {wiki?.title}
+              </Heading>
+            </Flex>
+            <Flex
+              display={{
+                base: 'block',
```
Comment (anubra266) [inline src/components/Wiki/WikiPage/WikiMarkup.tsx:84]:
```suggestion
            <chakra.div
              display={{
                base: 'block',

**Review by anubra266 — CHANGES_REQUESTED** [review]

**Review by Softdev1 — COMMENTED** [review]
Yeah, what @Emmanueldmlr  pointed out should be fixed actually, that is the only issue

**Comment by Royal-lobster** [comment]
> Yeah, what @Emmanueldmlr  pointed out should be fixed actually, that is the only issue 

@Softdev1 the bug is fixed in the previous commit, here is the updated url https://monopedia-ui-git-fix-wiki-layout-page-prediqt.vercel.app/wiki/remember-you-will-have-to-manually-add-that

**Review by Emmanueldmlr — APPROVED** [review]
Nice.

**Review by Softdev1 — APPROVED** [review]

**Pull request merged**